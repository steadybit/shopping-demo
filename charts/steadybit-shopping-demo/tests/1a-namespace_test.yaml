templates:
  - 1a-namespace.yaml
chart:
  appVersion: v0.0.0
tests:
  - it: manifest should match snapshot
    set:
    asserts:
      - matchSnapshot: { }