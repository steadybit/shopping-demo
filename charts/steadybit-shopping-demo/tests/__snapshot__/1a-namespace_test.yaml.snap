manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: Namespace
    metadata:
      labels:
        app: steadybit-demo
      name: steadybit-demo
